packages/cli/src/commands/generate/auth/providers/ethereum.js

```diff
@@ -0,0 +1,60 @@
+// the lines that need to be added to index.js
+export const config = {
+  imports: [
+    `import EthereumAuthClient from '@oneclickdapp/ethereum-auth'`,
+    `import { ApolloClient, InMemoryCache } from '@apollo/client'`,
+    `import { FetchConfigProvider, useFetchConfig } from '@redwoodjs/web'`,
+  ],
+  init: `const ApolloInjector = ({ children }) => {
+  const { uri, headers } = useFetchConfig()
+  let ethereum
+  try {
+    const graphQLClient = new ApolloClient({
+      cache: new InMemoryCache(),
+      uri,
+      headers,
+    })
+    // Default option using Apollo Client
+    const makeRequest = (mutation, variables) =>
+      graphQLClient.mutate({
+        mutation,
+        variables,
+      })
+
+    // Alternative option using graphql-hooks
+    // You'll also need to modify graphQLClient
+    // const makeRequest = (query, variables) =>
+    //   graphQLClient.request({
+    //     query,
+    //     variables,
+    //   })
+
+    ethereum = new EthereumAuthClient({
+      makeRequest,
+      // Note: you must set NODE_ENV manually when using Netlify
+      debug: process.NODE_ENV !== 'production',
```

Comment:
We've had some problems comparing `NODE_ENV` with `'production'`.
```suggestion
      debug: process.NODE_ENV === 'development',
```
This would make me feel safer